lib/routes/exposures/index.js

```diff
@@ -146,6 +201,25 @@ async function exposures(server, options, done) {
     }
   })
 
+  /**
+   * Allows users to request a list of all the exposure files that have been generated since they last
+   * requested exposure files. The caller includes `since` which is the largest exposure file ID they've
+   * seen. Exposure file IDs are sequentially generated by the database, and always count up. So if the
+   * user has seen files 1,2,3,4,5 then they'll pass `since = 5` and this will return a list of files
+   * with ID > 5.
+   *
+   * The user can also include a `limit` value to indicate the number of files they want to receive in
+   * the response. If they include `limit=2` then, using the above example, they SHOULD receive a response
```

Comment:
The limit parameter is no longer supported due to changes in file generation. Multiple files are now generated to allow users to catch up on days they may have missed by downloading a single file.

Reply:
Thanks @colmharte. I've updated the comments to include this change.